### PR TITLE
Update checking-mco-node-status.adoc

### PR DESCRIPTION
**Pull request by diksha1999**
Added feature gate in the prerequisites section

Version(s): 4.15 and 4.16
 

Issue: [OCPBUGS-37434](https://issues.redhat.com/browse/OCPBUGS-37434)

Link to docs preview:   https://github.com/diksha1999/openshift-docs/blob/main/modules/checking-mco-node-status.adoc

QE review:
- [ ] QE has approved this change.

Additional information: 
https://docs.openshift.com/container-platform/4.15/post_installation_configuration/machine-configuration-tasks.html#checking-mco-node-status_post-install-machine-configuration-tasks

### CODE REVIEW

File: modules/checking-mco-node-status.adoc [hunk 1]
```diff
@@ -13,6 +13,12 @@ To see the status of the Machine Config Operator (MCO) updates to your cluster,
 :FeatureName: Improved MCO state reporting
 include::snippets/technology-preview.adoc[]
 
+.Prerequisites
+* `TechPreviewNoUpgrade` feature set is enabled on your cluster.
+
+.Additional resources
+* See xref:../nodes/clusters/nodes-cluster-enabling-features.adoc[Enabling feature sets using the web console] for more information about feature gates.
```
Comment (snarayan-redhat) [inline modules/checking-mco-node-status.adoc:20]: This section needs to be in the assembly after this module. We do not place xrefs in modules.

[the remark this answers is not in the document]
Comment (snarayan-redhat) [reply modules/checking-mco-node-status.adoc:20]: This also needs to be addressed.

**Review by snarayan-redhat — COMMENTED** [review]
Nice work. Couple of comments.

**Comment by snarayan-redhat** [comment]
Couple more comments:

- Commits need to be squashed. 
- QE approval is required.

File: modules/checking-mco-node-status.adoc (hunk 1 above)
Comment (snarayan-redhat) [inline modules/checking-mco-node-status.adoc:20]:
```suggestion
* For more information about feature gates, see xref:../nodes/clusters/nodes-cluster-enabling-features.adoc[Enabling feature sets using the web console] .
```

**Comment by diksha1999** [comment]
Closing this PR due to issues; a new [PR 79681](https://github.com/openshift/openshift-docs/pull/79681) has been raised with the necessary changes.

**Comment by diksha1999** [comment]
Closing this one, opened a new [PR](https://github.com/openshift/openshift-docs/pull/79682) for making these changes.

**Pull request closed without merging**